app/repositories/address.js

```diff
@@ -0,0 +1,41 @@
+const db = require('../models');
+
+class AddressRepository {
+  getByAddress(address) {
+    return db.addresses.findOne({
+      where: { ...address },
+      paranoid: false,
+    });
+  }
+
+  get(id) {
```

Comment:
You can do the thing you did for the controllers here, for the repositories as well. You can have a base repository class where you implement the common methods and these specific repositories can extend the base one.